lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py

```diff
@@ -106,6 +116,23 @@
           disk:
             bus: virtio
 
+- name: Create virtual machine 'myvm' with multus network interface
+  kubevirt_vm:
+      name: myvm
+      namespace: vms
+      memory: 512M
+      interfaces:
```

Comment:
Is there a way to specific static network configuration such as IP or MAC addresses?

Reply:
Not any I am aware of, it would be possible in case of using multus and OVS, if you pass cloud_init specyfing the IP, but deffinitelly not by passing it in interface,

Reply:
OK